cmd/import/sample.sql

```diff
@@ -0,0 +1,755 @@
+/*
+Sample DB for the import command. You can create an sqlite DB with the command:
+
+$ sqlite3 sample.db < sample.sql
+*/
+CREATE TABLE files (name_a TEXT, name_b TEXT, content_a TEXT, content_b TEXT);
```

Comment:
Most probably, eventually we will want to keep more information on files i.e
 - `repository_id` they are coming from
 - `commit_hash` inside that repository
 - path
 - `blob_id` sha1 hash of the blob

But this can be taken care of in subsequent PRs.

Reply:
Yes, we can easily add that properties or they can even use the name for that (it would not require changes in our app), for example:
they can name the files as: `<repo>/blob/<commit>/<path>`, so it would include the repository, commit, and path, example:
`github.com/src-d/go-git/blob/110e7012e498c33ba2cbc866f60bb52edca947ae/README.md`
.

Reply:
I agree, but until we have real input examples, this file assumes the bare minimum data to start developing.